common/scala/build.gradle

```diff
@@ -10,6 +10,7 @@ repositories {
 
 dependencies {
     compile "org.scala-lang:scala-library:${gradle.scala.version}"
+    compile "org.scala-lang:scala-reflect:${gradle.scala.version}"
```

Comment:
As noted below, we can get around that dependency by using `MAnifest` over `ClassTag`